Modules/xxlimited_35.c

```diff
@@ -0,0 +1,301 @@
+
+/* This module is compiled using limited API from Python 3.5,
+ * making sure that it works as expected.
```

Comment:
If you prefer Python 3.5 over Python 3.6, maybe add a note that C99 is not used, since C99 is only supported since Python 3.6. I suggest that since I wanted to use you to defined types with C99 ".member = ..." syntax :-)